account/permissions.py

```diff
@@ -0,0 +1,6 @@
+from rest_framework import permissions
+
+
+class IsOwner(permissions.BasePermission):
+    def has_object_permission(self, request, view, obj):
+        return obj.owner == request.user and request.data.owner != request.user
```

Comment:
in permission b IsAuthenticated ke bala ye method miznim che farghi dare? to tele bego yedt mond